CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## UNRELEASED
+
+### Changed
+
+- The default value for ``default_ttl`` is changed from ``null`` to ``0``. 
```

Comment:
as null did not work as expected anyways, i think this change is ok. there only is a problem if somebody explicitly created the plugin with (null, false) - but imo that falls into the category of a bugfix that can be a BC break if you relied on the bug.